drivers/timer/Kconfig.nrf_rtc

```diff
@@ -23,6 +23,17 @@ config NRF_RTC_TIMER_USER_CHAN_COUNT
 	help
 	  Use nrf_rtc_timer.h API. Driver is not managing allocation of channels.
 
+config NRF_RTC_TIMER_LOCK_ZERO_LATENCY_IRQS
+	# hidden option
+	bool
+	depends on ZERO_LATENCY_IRQS
+	default y if !BT_LL_SW_SPLIT
```

Comment:
I don't follow why the protection against Zero Latency IRQs is not needed when `CONFIG_BT_LL_SW_SPLIT=y`. Could you please explain?

Reply:
Zephyr Bluetooth Controller does not have any callbacks from Zero Latency ISRs when Zero Latency IRQs are enabled, ruling out applications from using kernel features or calling timer APIs. Also, Zephyr Bluetooth Controller can operate using normal IRQs while applications can use Zero Latency IRQs for their sensors (as long as latencies are under control).

Refer to  a customer discussion here: https://github.com/zephyrproject-rtos/zephyr/discussions/41401, in the end Zero Latency IRQs where not required to be used with Zephyr Bluetooth Controller.